data/en/isjson.json

```diff
@@ -17,6 +17,15 @@
 	},
 	"links": [
 
+	],
+	"examples":[
+	{
+		"title":"isJSON Example",
+		"description":"Returns true if the parameter is a valid JSON value. Returns False if the parameter is not a valid JSON data representation.",
+		"code":"isJSON("[1,2,3]")",
```

Comment:
Need to escape the quotes in the code example... so it should be:

`"code":"isJSON(\"[1,2,3]\")",`